### PR TITLE
Snap sheet components to 4px grid

### PR DESCRIPTION
**Pull request by sophialee0416**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Some components do not snap to a 4px grid. This change should fix Sheets, specifically:

1. Bottom sheet

- With one-line list
- With one-line list with icons
- With header and one-line list with icons
- With description and one-line list with icons

2. Top sheet

- Default
- With one-line list

3. Floating list

- With one-line list

### Verification

Measured new components with grids and through debugging.

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-27 at 19 09 25 | Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-27 at 19 08 28 |
| Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-27 at 19 06 14 | Simulator Screen Shot - iPhone 11 Pro Max - 2020-10-27 at 19 07 22 |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [x] Different resolutions (1x, 2x, 3x)
- [x] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/277)

### CODE REVIEW

File: ios/FluentUI/Popup Menu/PopupMenuItemCell.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
     }
 
     private struct Constants {
-        static let labelVerticalMarginForOneLine: CGFloat = 14
+        static let labelVerticalMarginForOneLine: CGFloat = 15
```
Comment (Saadnajmi) [inline ios/FluentUI/Popup Menu/PopupMenuItemCell.swift:17]: This doesn't need to be like a multiple of 4?

Comment (sophialee0416) [reply ios/FluentUI/Popup Menu/PopupMenuItemCell.swift:17]: The total pixel height of the cell was originally 50px, this addition makes it 52px

Comment (harrieshin) [reply ios/FluentUI/Popup Menu/PopupMenuItemCell.swift:17]: my math doesn't add up tho?

Comment (sophialee0416) [reply ios/FluentUI/Popup Menu/PopupMenuItemCell.swift:17]: The text area height is currently at 22, + vertical margin top and bottom = 52

**Review by harrieshin — APPROVED** [review]

**Pull request merged**